db.go

```diff
@@ -752,9 +779,6 @@ func (db *DB) recover(ctx context.Context, wal WAL) error {
 	// WAL (i.e. entries that occupy space on disk but are useless).
 	performSnapshot := false
 
-	// Writes are performed concurrently to speed up replay.
-	var writeWg errgroup.Group
-	writeWg.SetLimit(runtime.GOMAXPROCS(0))
```

Comment:
It's written in the commit message but it's because an insert is an extremely fast now, as it's just an insert into the linked list. The concurrency made sense with the btree